src/parsers/manifest/utils/check_manifest_ids.ts

```diff
@@ -27,7 +27,7 @@ import { IParsedManifest } from "../types";
  * @param {Object} manifest
  */
 export default function checkManifestIDs(
-  manifest : IParsedManifest
+  manifest : { periods: Array<{ adaptations: IParsedAdaptations }> }
```

Comment:
I just thought about something... We never check the period's ID? Why? Do they exist?